frame/balances/src/lib.rs

```diff
@@ -277,6 +409,87 @@ decl_error! {
 		/// Beneficiary account must pre-exist
 		DeadAccount,
 	}
+
+	/// The total units issued in the system.
+	#[pallet::storage]
+	#[pallet::getter(fn total_issuance)]
+	pub type TotalIssuance<T: Config<I>, I: 'static = ()> = StorageValue<_, T::Balance, ValueQuery>;
+
+	/// The balance of an account.
+	///
+	/// NOTE: This is only used in the case that this module is used to store balances.
+	#[pallet::storage]
+	pub type Account<T: Config<I>, I: 'static = ()> = StorageMap<
+		_,
+		Blake2_128Concat,
+		T::AccountId,
+		AccountData<T::Balance>,
+		ValueQuery
+	>;
+
+	/// Any liquidity locks on some account balances.
+	/// NOTE: Should only be accessed when setting, changing and freeing a lock.
+	#[pallet::storage]
+	#[pallet::getter(fn locks)]
+	pub type Locks<T: Config<I>, I: 'static = ()> = StorageMap<
+		_,
+		Blake2_128Concat,
+		T::AccountId,
+		Vec<BalanceLock<T::Balance>>,
+		ValueQuery
+	>;
+
+	/// Storage version of the pallet.
+	///
+	/// This is set to v2.0.0 for new networks.
+	#[pallet::storage]
+	pub(super) type StorageVersion<T: Config<I>, I: 'static = ()> = StorageValue<
+		_,
+		Releases,
+		ValueQuery
+	>;
+
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config<I>, I: 'static = ()> {
+		pub balances: Vec<(T::AccountId, T::Balance)>,
+	}
+
+	#[cfg(feature = "std")]
+	impl<T: Config<I>, I: 'static> Default for GenesisConfig<T, I> {
+		fn default() -> Self {
+			Self {
+				balances: Default::default(),
+			}
+		}
+	}
+
+	#[pallet::genesis_build]
+	impl<T: Config<I>, I: 'static> GenesisBuild<T, I> for GenesisConfig<T, I> {
```

Comment:
pallet balances is used by many other pallets, so I guess their tests will fail to find `assimilate_storage`. Either fix the tests to import `GenesisBuild` or do same as you did in frame_system (I.e. implement them manually again).

Reply:
You beat the compiler to telling me this :crab: